### ISSUE TITLE
Transfer Learning with Custom Pretrained model

### ISSUE THREAD

**Issue by tugbakara** [ultralytics/yolov5#11756]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

hey, I have dataset with 7 different classes and I have roboflow pretrained model with 2 labels different than dataset's labels. I want to merge them with transfer learning and I want to use only dataset's labels in new model. In that case , using roboflow model will improve and be affective in new model or will this just be normal training a model ? 
Also I have different size images in my dataset and model trained with different sizes, will this affect in a bad way a new model and lastly, is there any tutorial can I follow to achieve transfer learning purpose?  I read dozens of issue in github and question in forum but I saw some frozening layers etc.. It looks confused. Also which changes do I need to apply in config , yaml files?

EDIT:
here is the most detailed link to explain myself [1](https://community.ultralytics.com/t/transfer-learning-with-custom-pretrained-model/601) , [2](https://community.ultralytics.com/t/how-to-combine-weights-to-detect-from-multiple-datasets/38/62?u=tugbakara)

**Comment by glenn-jocher** [comment]
@tugbakara hi there! It's great to see your interest in customizing a pre-trained YOLOv5 model for your specific dataset. Utilizing transfer learning with your custom pretrained model can definitely be beneficial in improving and accelerating the training of your new model. 🚀

When merging with transfer learning, you can discard the pretrained model's unwanted labels and train exclusively on your dataset's labels. The different sizes of the images in your dataset and the pretrained model won't negatively impact the new model, as YOLOv5 is designed to handle varying input sizes efficiently. As for tutorials, our documentation provides a detailed guide to achieve your transfer learning goals. You might find it helpful to check out the Custom YOLOv5 Models section.

In terms of config and yaml files, you can specify your dataset's classes and update parameters like image size and batch size to align with your specific requirements.

If you encounter any challenges along the way, don't hesitate to reach out and we'll be happy to assist! Good luck with your transfer learning journey!
